### PR TITLE
fix(validate): validate STAC Item against item-spec json schema (TDE-204)

### PR DESCRIPTION
**Pull request by paulfouquet**

### CODE REVIEW

File: topo_processor/stac/item.py [hunk 1]
```diff
@@ -30,6 +31,7 @@ def __init__(self, item_id: str):
         self.collection = None
         self.geometry_poly = None
         self.assets = []
+        self.schema = 'https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json'
```
Comment (blacha) [inline topo_processor/stac/item.py:34]: Is this defined in pystac anywhere? could we just reference it from there?

Comment (paulfouquet) [reply topo_processor/stac/item.py]: I've pushed a change this morning

**Review by amfage — APPROVED** [review]

File: topo_processor/stac/item.py (hunk 1 above)
Comment (blacha) [reply topo_processor/stac/item.py]: Wow that is kinda frustrating how hard it is to get a schema URI from pystac!  maybe we do just want to hard code them :P

**Review by blacha — APPROVED** [review]

File: topo_processor/stac/item.py (hunk 1 above)
Comment (paulfouquet) [reply topo_processor/stac/item.py]: I know right!

**Pull request merged**